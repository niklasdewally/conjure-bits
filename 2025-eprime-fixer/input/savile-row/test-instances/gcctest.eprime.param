language ESSENCE' 1.0

letting v be [2,4,5,6,7]

