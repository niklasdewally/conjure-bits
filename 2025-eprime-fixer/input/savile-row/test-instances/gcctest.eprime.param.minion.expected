MINION 3

#This file has been automatically generated by TAILOR v0.3.3
#bug-reports: andrea@cs.st-and.ac.uk

**VARIABLES**
DISCRETE m[10] {1..10}

# auxiliary variables
BOOL aux0 
BOOL aux1 
BOOL aux2 
BOOL aux3 

# aliases

**SEARCH**

PRINT [
m]

VARORDER [
m,
aux0,aux1,aux2,aux3]


**CONSTRAINTS**

reify(eq(5, m[0]), aux0)
reify(gcc([m], [1,2,3], [2,2,2]), aux1)
reify(gcc([m], [2,4,5,6,7], [2,2,2,2,2]), aux2)
reify(sumgeq([aux2,aux1], 1), aux3)
ineq(aux0,aux3,0)
gcc([m], [1,2,3], [2,2,2])
gcc([m], [1,2,3,4,5,6,7,8,9,10], [m])

**EOF**
